uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1020.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1020 {
+
+  /**
+   * Rename Line Manager Relationship
+   */
+  public function upgrade_1020() {
+    $this->up1020_renameLineManagerRelationship();
+
+    return TRUE;
+  }
+
+  /**
+   * Renaming Line Manager Relationship
+   */
+  private function up1020_renameLineManagerRelationship() {
```

Comment:
Same as my suggestion for the PR title, if it's just the label change please rename this function and update the docblock